gitignore/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+ 
```

Comment:
- [ ] Good job working on the "Portfolio: setup and mobile first" 😃, but currently, your project doesn't have any `index.html` and `css` files 😸. Kindly make sure you create them and customize your project according to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=rHQ6yd5LRBvNFkFE-0) given, and make sure your website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.).

## See the  [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) 😃